### DISCUSSION TITLE
How do I fix container labels auto reset while saving?

### DISCUSSION THREAD

**Discussion by alexzvn** [coollabsio/coolify#2548] [Q&A] [answered]
Every time i try to  save new configuration and it wipeout.

https://github.com/coollabsio/coolify/assets/41188285/3b7fdd88-735d-461a-b2af-55dba6a26eac

**Comment by softwaretirol** [comment]
Same problem - downgraded some version and it works again. 294 is currently not stable for that.

**Reply by alexzvn** [reply]
which version you're running?

**Reply by devdjdjdj** [reply]
Same problem, I am running 4.0.0 beta 297

**Comment by alexzvn** [comment, marked as answer]
Okay, this bugs come with feature set redirect `www` and `non-www`.
I'll make a [PR](https://github.com/coollabsio/coolify/pull/2553) for it.
